red-hat-data-services/odh-dashboard
# Add UI for Bring Your Own Notebook Image [BYON]

- [ ] JIRA link(s): https://issues.redhat.com/browse/RHODS-3059

TEST STEPS:
1. Login to the Dashboard as a user in the RHODS admin group `dedicated-admins`
2. Verify `Settings` -> `Notebook Images` is enabled
3. Click on `Notebook Images`
4. Click `Import Image` and specify any valid notebook image repo url, notebook name with OPTIONAL description, software and packages
5. Click `Import`
6. Verify that the Imagestream and tag was created
7. Login to JH and spawn a notebook with that custom image
